code/modules/mob/living/carbon/xenomorph/castes/wraith/castedatum_wraith.dm

```diff
@@ -123,11 +123,11 @@
 	plasma_gain = 25
 
 	// *** Health *** //
-	max_health = 260
+	max_health = 310
```

Comment:
```suggestion
	max_health = 310
```
a 50 point increase to a caste with decent armor stats and incredible mobility outisde of being stagger seems kinda high, i'd go with capping out at 290

Reply:
50 Point increase is because its a T2 the health is normal for a T2. It's non bullet armor is dogshit so dno where you getting decent armor stats from.